Python-Home-Challenges/Creating_file.sh

```diff
@@ -0,0 +1,3 @@
+cd /
```

Comment:
it is not a good idea to write file on linux (or any os) root, use /tmp or /home/$USER/ instead